tests/AccountOAuthTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Stripe;
```

Comment:
@ob-stripe  I had to create a new file because PHPUnit seems to require one class per file otherwise it just ignores the other ones.

Reply:
Ack. I'll extend `stubRequest` to be able to handle non-`api.stripe.com` requests and we'll be able to move the test back in `AccountTest`.

Reply:
I pushed another commit to move this test directly in `AccountTest`.